sdk/storage/Azure.Storage.Blobs/tests/AppendBlobClientTests.cs

```diff
@@ -49,6 +49,27 @@ public void Ctor_ConnectionString()
             Assert.AreEqual("accountName", builder.AccountName);
         }
 
+        [Test]
```

Comment:
Please make sure all kinds of blob client can be created with all kinds credential, and have test cases to cover them.

Reply:
I did.

Reply:
There are test cases for BlobClient, BlobBaseClient, AppendBlobClient, and PageBlobClient.

Reply:
Please confirm  sure all kinds  of credential are covered, and BlockBlobClient also covered.

Reply:
https://github.com/Azure/azure-sdk-for-net/pull/13332/files#diff-85f647442c3573f2903357a126170dacR60

Reply:
Hi Sean, 
The above link is for BlockblobClient.
Please confirm all kinds of credentail are covered by all kinds of blob clients.
(I see the test for Uri only, please make sure oauth, key, connection string are all covered)